.github/workflows/promote.yaml

```diff
@@ -8,21 +8,35 @@ on:
       tag_name:
         description: 'Name for the tag of the release.'
         required: true
+      quay_registry:
+        description: 'Quay registry to push to.'
+        default: 'quay.io/ansible'
+
+env:
+  QUAY_REGISTRY: ${{ vars.QUAY_REGISTRY }}
 
 jobs:
   promote:
     runs-on: ubuntu-latest
     steps:
-      - name: Set TAG_NAME for workflow_dispatch event
+      - name: Set GitHub Env vars for workflow_dispatch event
         if: ${{ github.event_name == 'workflow_dispatch' }}
         run: |
           echo "TAG_NAME=${{ github.event.inputs.tag_name }}" >> $GITHUB_ENV
+          echo "QUAY_REGISTRY=${{ github.event.inputs.quay_registry }}" >> $GITHUB_ENV
 
-      - name: Set TAG_NAME for release event
-        if: ${{ github.event_name == 'release' }}
+      - name: Set GitHub Env vars if release event and vars.QUAY_REGISTRY is not set
+        if: ${{ github.event_name == 'release' && !vars.QUAY_REGISTRY }}
```

Comment:
why?

Reply:
vars.QUAY_REGISTRY could be set and we still need to set the TAG_NAME

Reply:
This was a mistake, I just removed that conditional